api-docs/docs/index.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 1
+---
+
+# JavaScript Trackers
+The Snowplow JavaScript Trackers support being used in a number of environments. There are three versions of the tracker.
+
+## Web tracking
+We have two flavours of web tracker: JavaScript and Browser.
+
+- JavaScript Tracker (v2 and v3) for loading via tags, by adding code snippets to your website or Tag Manager solution.
+- Browser Tracker (v3) for installation into web apps via npm. Popular when natively integrating tracking into React, Angular and Vue applications.
```

Comment:
Can we add some links here to the JS docs pages to guide users to a set-up guide for the Web and NodeJS trackers?